doc/WorldBuilding.md

```diff
@@ -36,7 +42,7 @@ Now your customised start, end and level progression screens will be used in you
 
 3. Open the first level in your world in the Editor's 2D view.
 
-4. Select the `TileMapLayer` from the Scene Dock. In the Inspector, locate the `Tile Set` option under `TileMapLayer`.
+4. Select the `Map` node from the Scene Dock. In the Inspector, locate the `Tile Set` option under `Player`.
```

Comment:
Was the 2nd change accidental? It's still under TileMapLayer, not Player.
![image](https://github.com/user-attachments/assets/4f18b6d2-e687-4c85-a352-7ac1ed3fb449)

Reply:
Yes, it was accidental. Good catch.